source/validate.h

```diff
@@ -190,4 +190,11 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInstructions,
                             spv_position position,
                             const spvtools::MessageConsumer& consumer);
 
+// Performs validation for the SPIRV-V module binary and returns the
+// ValidationState to the caller.
```

Comment:
Well, it just returns a status.  If you want to say "return" the maybe return an actual pair of status and unique-ptr.